Libraries/Hotcakes.Commerce.Dnn/Hotcakes.Commerce.Dnn.csproj

```diff
@@ -270,6 +271,14 @@
   </ItemGroup>
   <ItemGroup />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <Target Name="EnsureNuGetPackageBuildImports" BeforeTargets="PrepareForBuild">
```

Comment:
This has the same concern as my first comment.  However, if there is a good technical reason, then this will be approved.

Reply:
I will make a new update with a cleaner solution than this. As I have mentioned, the changes were added automatically when using the Nuguet-Package-Manager tool.

Reply:
@ArielBlanco1990 Okay. I figured that was the case, but let's make sure these updates are absolutely necessary.  Thank you!  :)

Reply:
This pull request needs some changes before being approved.